chipy_org/apps/job_board/templates/job_board/job_post_detail.html

```diff
@@ -2,46 +2,42 @@
 
 {% block body %}
 
+<a href="?show_filled=true">Show Filled Jobs</a>
+
+<ul>
+    {% for job in jobs %}
+        <li>{{ job.title }} - 
+            {% if job.is_filled %}
+                Filled on {{ job.filled_date }}
+            {% else %}
+                Open
+            {% endif %}
+        </li>
+    {% endfor %}
+</ul>
```

Comment:
I don't think this is what we want here.  This is the job post details page and should be related to a single post.  So I wouldn't expect a loop over jobs. Something like:

<p>
  <b>
     Available: 
        {% if job.is_filled %} 
           Filled on {{ job.filled_date }} 
       {% else %} 
           Yes 
        {% endif %}
   </b>
</p>

Though I'm still unsure why we would list filled jobs ... :thinking:

Reply:
Okay I get this now, but there's no limit or pagination here. What if we have a 1000+ filed jobs?   Unlikely I know, but we should put some limits like 10 or a pagination.  

Let's maybe just make this out of scope and remove it.